### PR TITLE
No errors showing. Using Configurable trait now.

### PR DESCRIPTION
**Pull request by mitchfaber**

### CODE REVIEW

File: _config/config.yml [hunk 1]
```diff
@@ -7,5 +7,5 @@ SilverStripe\Core\Injector\Injector:
         constructor:
             namespace: "NewRelic"
             defaultLifetime: 86400 #Cache time in seconds
-PackagistShortCode:
+WebbuildersGroup\PackagistShortCode:
```
Comment (UndefinedOffset) [inline _config/config.yml]: This is incorrect it should be `WebbuildersGroup\PackagistShortCode\PackagistShortCode`

**Review by UndefinedOffset — CHANGES_REQUESTED** [review]
It looks like a number of files came from the webbuilders-group/silverstripe-githubshortcode module, why did these get added? There should be equivalents of these files in the module.

File: src/PackagistShortCode.php [hunk 2]
```diff
@@ -4,113 +4,120 @@
...
+        $obj->MonthlyDownloads = $cachedData['monthly'];
+        $obj->DailyDownloads = $cachedData['daily'];
+
+
         //Init ss viewer and render
-        Requirements::css(PACKAGISTSHORTCODE_BASE.'/css/PackagistButton.css');
-        
-        $ssViewer=new SSViewer('PackagistButton');
-        return $ssViewer->process($obj);
+        Requirements::css('webbuilders-group/silverstripe-githubshortcode:css/GitHubButtons.css');
```
Comment (UndefinedOffset) [inline src/PackagistShortCode.php]: This should be `webbuilders-group/silverstripe-packagistshortcode:css/PackagistButton.css`

File: src/PackagistShortCode.php [hunk 3]
```diff
@@ -4,113 +4,120 @@
...
+
+        // $ssViewer = new SSViewer('PackagistButton');
+        // return $ssViewer->process($obj);
```
Comment (UndefinedOffset) [inline src/PackagistShortCode.php]: Something tells me with this removed this does not render at all

**Review by UndefinedOffset — CHANGES_REQUESTED** [review]
Can you also update the README.md in the root (the configuration options section to be specific) just needs the namespace added.

File: src/PackagistShortCode.php (hunk 3 above)
Comment (mitchfaber) [reply src/PackagistShortCode.php]: Indeed! I thought these lines were doing something else. Fixed in the next commit.

File: README.md [hunk 4]
```diff
@@ -50,6 +50,6 @@ SilverStripe\Core\Injector\Injector:
         constructor:
             namespace: "NewRelic"
             defaultLifetime: 86400 #Cache time in seconds
-PackagistShortCode:
+WebBuildersGroup\PackagistShortCode\PackagistShortCode:
```
Comment (UndefinedOffset) [inline README.md]: So minor... but it will cause issues `WebBuildersGroup` must match the case defined as the namespace `WebbuildersGroup`

**Review by UndefinedOffset — CHANGES_REQUESTED** [review]

**Review by UndefinedOffset — APPROVED** [review]

**Pull request merged**